src/test/Microsoft.Extensions.DependencyModel.Tests/DependencyContextJsonReaderTest.cs

```diff
@@ -30,7 +28,7 @@ public void ReadsRuntimeTargetInfo()
         ""signature"":""target-signature""
     },
     ""targets"": {
-        "".NETCoreApp,Version=v1.0/osx.10.10-x64"": {},
+        "".NETCoreApp,Version=v1.0/osx.10.10-x64"": {}
```

Comment:
What's really weird is that using DependencyModel `v2.1`, I tried adding trailing commas and I was getting exceptions being thrown from the current implementation. I'm surprised that these tests are passing.